common/autoware_auto_geometry/CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ ament_auto_add_library(${PROJECT_NAME} SHARED
   src/spatial_hash.cpp
   src/bounding_box.cpp)
 autoware_set_compile_options(${PROJECT_NAME})
+# workaround to allow deprecated header to build on both galactic and rolling
+target_compile_options(${PROJECT_NAME} PUBLIC
+  -Wno-error
```

Comment:
Please apply the same solution as in https://github.com/autowarefoundation/autoware.universe/pull/720#discussion_r852040761